core/frontend/src/components/scanner/servicesScanner.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <span />
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import axios from 'axios'
+import { getModule } from 'vuex-module-decorators'
+import ServicesScannerStore from '@/store/servicesScanner'
+import {Service} from '@/types/SERVICE'
+
+const servicesHelper: ServicesScannerStore = getModule(ServicesScannerStore)
+
+/**
+ * Actual scanner for running services.
+ * This periodically fetches /helper/latest/web_services
+ * and updates the ServiceHelperStore with the available services
+ * @displayName Services Scanner
+ */
+export default Vue.extend({
+  name: 'ServicesFetcher',
+
+  mounted () {
+    // Fetch network data
+    this.requestData()
+
+    // Fetch periodic API request
+    this.startPeriodicRequest()
+  },
+  methods: {
+    startPeriodicRequest () {
+      setInterval(() => {
+        this.requestData()
+      }, 5000)
+    },
+    requestData () {
+      axios.get('http://localhost/helper/latest/web_services').then((response) => {
+        // Sort services by port number
+        const services = response.data.sort(
+          (first: Service, second: Service) => first.port - second.port,
+        )
+        servicesHelper.updateFoundServices(services)
+      }).catch((error) => {
+        console.log('Error scanning for services:')
```

Comment:
Maybe:
```suggestion
        console.log(`Error scanning for services: ${error}`)
```

Reply:
the explanation is in the next line. doing this prints the error as a string, logging int alone give us better information.

Reply:
About that, I'm not sure if both lines will run without interruption of another `console.log` when a interruption/event/async happens. But I'm ok if you think that the current approach should work.

Reply:
I really prefer the extra details we get from logging the error on its own.

Reply:
Does `console.log('Error scanning for services: ', error)` works for you ?

Reply:
yeah, that works. updated.